Sources/PointFree/GitHub.swift

```diff
@@ -107,5 +107,15 @@ private func runGitHub<A>(_ gitHubRequest: DecodableRequest<A>) -> EitherIO<Erro
   return jsonDataTask(with: gitHubRequest.rawValue, decoder: gitHubJsonDecoder)
 }
 
+private let gitHubJsonEncoder: JSONEncoder = { () in
+  let encoder = JSONEncoder()
+
+  if #available(OSX 10.13, *) {
+    encoder.outputFormatting = [.sortedKeys]
```

Comment:
I think this lil bit of extra work is fine to do in production, but if ya wanna move it into `Current` lemme know!

Reply:
yeahhhhh that's fine I think.